bundle/metadata/annotations.yaml

```diff
@@ -1,12 +1,15 @@
 annotations:
-  operators.operatorframework.io.bundle.channel.default.v1: 1.x
-  operators.operatorframework.io.bundle.channels.v1: alpha,1.x
-  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  # Core bundle annotations.
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: kogito-operator
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.2.0
+  operators.operatorframework.io.bundle.channels.v1: alpha,1.x
+  operators.operatorframework.io.bundle.channel.default.v1: 1.x
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.10.0+git
```

Comment:
same here

Reply:
operator-sdk-v1.10.0+git label is for 1.11.0